pkg/action/executor/machine_scale_executor.go

```diff
@@ -45,15 +45,14 @@ func (s *MachineActionExecutor) Execute(vmDTO *TurboActionExecutorInput) (*Turbo
 	case proto.ActionItemDTO_PROVISION:
 		actionType = ProvisionAction
 		diff = 1
-		break
 	case proto.ActionItemDTO_SUSPEND:
 		actionType = SuspendAction
 		diff = -1
-		break
 	default:
 		return nil, fmt.Errorf("unsupported action type %v", vmDTO.ActionItems[0].GetActionType())
 	}
 	// Get on with it.
+	glog.V(3).Infof("Starting execution of machine scaling action: %+v on node %v.", actionType, nodeName)
```

Comment:
Suggest:

* V(2)
* Depending on the action type, can we say something like the following:
  `Starting to scale up the machineset xxx by 1 replica`, or
  `Starting to scale down the machineset xxx by 1 replica`

Reply:
Done